Yubico.YubiKey/src/Yubico/YubiKey/Cryptography/AesUtilities.cs

```diff
@@ -179,29 +178,24 @@ public static byte[] AesCbcDecrypt(byte[] key, byte[] iv, ReadOnlySpan<byte> cip
                 throw new ArgumentException(ExceptionMessages.IncorrectCiphertextLength, nameof(ciphertext));
             }
 
-            byte[] plaintext;
+            using var aesObj = CryptographyProviders.AesCreator();
+            byte[] aesObjKey = key.ToArray();
+            byte[] aesObjIv = iv.ToArray();
```

Comment:
In the previous method you zero similar object before the method leaves, but not in this method.